rasa_sdk/knowledge_base/storage.py

```diff
@@ -110,6 +109,15 @@ async def get_object(
         """
         raise NotImplementedError("Method is not implemented.")
 
+    def get_object_types(self) -> List[Text]:
+        """
+        Returns a list of object types from knowledge base data.
+
+        Returns: the list of object types
+
```

Comment:
This repeats the above info, you can remove this and keep one line for the docstring.

Reply:
Done 👍🏽